base/common/src/org/dogtagpki/tps/apdu/GetDataAPDU.java

```diff
@@ -58,7 +58,7 @@ public TPSBuffer getEncoding()
         encoding.add(ins);
         encoding.add(p1);
         encoding.add(p2);
-        encoding.add((byte) 0x2D);
+        encoding.add((byte) 0x00);
```

Comment:
will the old cards work fine with this change?

Reply:
Yes, they have tested with legacy tokens.